### PR TITLE
feat: add weekly daily map layers

### LINKED ISSUE #21: Map Feature: Weekly/Daily map layers
[ ] Figure out how we want to handle rendering different map layers 
  - Investigate: these are all going to be geojson layers, with features that all look the same, so it probably won't even require two layers... what would be the benefit of two layers?

[ ] Add check boxes (denoting active layers) to the tool panel - this week/today quakes - maybe have widget on the map that shows the time frame of events we're viewing?

### PR DESCRIPTION
**Pull request by ash-bergs**
- Adds `layers` directory in components, and adds a layer to render the weekly and daily earthquake events 
- Adds `activeLayersAtom` to keep track of what layers are turned on 
- Adds geoJSON atoms for the weekly and daily events, getting the events from the relevant daily and weeklyEventsAtom and returning it as a Feature Collection so it can be used by the map layers 
- Adds basic layer toggles in the ToolPanel
- Both layers can be turned on at the same time, and both layers get the hover popup and ability to click and store events in the tool panel sidebar 

image

image

### CODE REVIEW

File: app/components/ToolPanel/ToolPanel.tsx [hunk 1]
```diff
@@ -28,7 +37,39 @@ const ToolPanel = () => {
...
+              });
+            }}
+            className={`flex items-center gap-2 p-2 ${
+              activeLayers.weekly ? 'bg-white' : 'bg-gray-200'
+            } rounded-md`}
+          >
+            Weekly
+            {activeLayers.weekly ? <FaEye /> : <FaEyeSlash />}
+          </button>
+        </div>
```
Comment (ash-bergs) [inline app/components/ToolPanel/ToolPanel.tsx:72]: image

These buttons are super basic - I haven't fully fleshed out this tool panel and I think it would be cool to have selections for daily/weekly, and the different magnitudes belonging to each. 

TLDR; These will change in appearance

File: store/index.ts [hunk 2]
```diff
@@ -27,8 +25,15 @@ export const mapRefAtom = atom<MapRef | null>(null);
 export const useMap = () => useAtomValue(mapRefAtom);
 export const setMap = () => useSetAtom(mapRefAtom);
 
-// Sneaky helpers
+/** Map Layer Configuration */
+// I see this getting really out of hand if the number of layers were to scale...
+export const activeLayersAtom = atom<any>({
+  daily: true,
+  weekly: false,
+});
```
Comment (ash-bergs) [inline store/index.ts:34]: Before we add layers for different magnitudes we really need to engineer a better data structure for this atom. 

So it be one atom `get`ting the value of several other?

File: app/components/ToolPanel/ToolPanel.tsx (hunk 1 above)
Comment (ash-bergs) [reply app/components/ToolPanel/ToolPanel.tsx:72]: Right now the DAILY shows *all* events, regardless of magnitude 

WEEKLY show those of 2.5 magnitude or higher 

Because I am going to plan out how I want to handle layers of varying magnitude - I think for now just adding some text to let the use know this is appropriate before merging.
